src/test/fuzz/load_external_block_file.cpp

```diff
@@ -15,7 +15,7 @@
 
 void initialize_load_external_block_file()
 {
-    InitializeFuzzingContext();
+    static TestingSetup setup{CBaseChainParams::REGTEST, {"-nodebuglogfile"}};
```

Comment:
In commit "fuzz: Initialize a full TestingSetup instead of BasicTestingSetup" (364fa81a2b014bd6257e011e6a201a48c1b8853e)

This seems safe and fine, but I'm only casually familiar with the fuzzing code and can't say I understand what the goal of the change and new code actually is. It might be good to have a code comment saying what the purpose of TestingSetup/REGTEST/nodebuglogfile is here, or a commit description saying what was bad about the previous InitializeFuzzingContext call

Reply:
Added some context in the commit message here: 4c4eb967016d80247cbf38e6811c8640761ada42
Lmk if that needs additional explanation!